### ISSUE TITLE
Availability of go-crud-api

### ISSUE THREAD

**Issue by tmetadev** [mevdschee/go-crud-api#19]
Was this port to Go ever finished? Just checking if you did more work on this Maurits.

Thank you.

**Comment by mevdschee** [comment]
No.. it is not finished at all. I implemented the core and never continued. If you want a full-featured implementation you should use the PHP version. The Java version is also nice and covers about 80% of the functionality, but I'm not aware of people using the Java version in production (as opposed to the PHP version that is quite popular).
